usr/local/share/bastille/update.sh

```diff
@@ -47,17 +47,42 @@ if [ $# -gt 1 ] || [ $# -lt 1 ]; then
     usage
 fi
 
-RELEASE="${1}"
+TARGET="${1}"
 shift
 
 if [ ! -z "$(freebsd-version | grep -i HBSD)" ]; then
     echo -e "${COLOR_RED}Not yet supported on HardenedBSD.${COLOR_RESET}"
     exit 1
 fi
 
-if [ -d "${bastille_releasesdir}/${RELEASE}" ]; then
-    freebsd-update -b "${bastille_releasesdir}/${RELEASE}" fetch install --currently-running "${RELEASE}"
+if [ -d "${bastille_jailsdir}/${TARGET}" ]; then 
+    if ! cat "${bastille_jailsdir}/${TARGET}/fstab" 2>/dev/null | grep -w "${TARGET}" | grep -q ".bastille"; then
```

Comment:
This can be simplified into something like: `grep -w "${TARGET}" "${bastille_jailsdir}/${TARGET}/fstab" | grep -q ".bastille"`

Reply:
Hi, we can grep directly on the target/fstab which is even simpler:
`grep -qw ".bastille" "${bastille_jailsdir}/${TARGET}/fstab"`

I will test and update PR today